examples/cfq/train.py

```diff
@@ -105,6 +107,27 @@ def pad_along_axis(array: jnp.array,
   return padded
 
 
+def compute_perfect_match_accuracy(predictions: jnp.array,
+                                   labels: jnp.array,
+                                   lengths: jnp.array) -> jnp.array:
+    """Compute perfect match accuracy.
+    
+    The accuracy is 1 only if the sequences are equal, otherwise is 0. The
```

Comment:
This explanation makes it sound like this function will only return 0 or 1, but in fact it will return the average accuracy over all inputs. Perhaps you can clarify this?